src/Internal/Service/PaymentProcessingService.php

```diff
@@ -45,22 +48,41 @@ class PaymentProcessingService {
 	 */
 	private $context_logger_service;
 
+	/**
+	 * Logger instance.
+	 *
+	 * @var Logger
+	 */
+	private $logger;
```

Comment:
The logger is not used here anymore.

Reply:
It's pretty easy to add it later if needed, but I don't think we will. AFAIAC, the service will not use the logger directly, and rely on `PaymentContextLoggerService` instead.